From 2b85ced4f4720be987032585b2b67d71983a889a Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Wed, 6 Nov 2013 19:14:08 +0800
Subject: [PATCH] ENGR00286724-9 dts: imx6qdl: add properties for flexcan to
 support wakeup

According to binding doc, add missed properties for remote wakeup
support.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
(cherry picked from commit 359b5a26f40200e8a9ed642d22e97d301f1c9d5c)
---
 arch/arm/boot/dts/imx6qdl.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl.dtsi b/arch/arm/boot/dts/imx6qdl.dtsi
index 03db334..a0d8bf5 100644
--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -17,6 +17,8 @@
 
 / {
 	aliases {
+		flexcan0 = &flexcan1;
+		flexcan1 = &flexcan2;
 		gpio0 = &gpio1;
 		gpio1 = &gpio2;
 		gpio2 = &gpio3;
@@ -415,6 +417,7 @@
 				clocks = <&clks IMX6QDL_CLK_CAN1_IPG>,
 					 <&clks IMX6QDL_CLK_CAN1_SERIAL>;
 				clock-names = "ipg", "per";
+				gpr = <&gpr>;
 				status = "disabled";
 			};
 
@@ -425,6 +428,7 @@
 				clocks = <&clks IMX6QDL_CLK_CAN2_IPG>,
 					 <&clks IMX6QDL_CLK_CAN2_SERIAL>;
 				clock-names = "ipg", "per";
+				gpr = <&gpr>;
 				status = "disabled";
 			};
 
-- 
1.8.0

